
<launch>

	<arg name="spin_rate" default="50" />
	<arg name="camera_topic" default="/camera/depth_registered/points" />
	<arg name="down_sample_size" default="0.01" />
	<arg name="pc_save" default="0" />
	<arg name="cylinders_topic" default="cylinders" />
	<arg name="camera_frame" default="camera_rgb_optical_frame" />

	<node name="desperate_house_wife_demo" pkg="desperate_housewife" type="phobic_camera" respawn="false" output="screen" >
 		
		<param name="camera_topic" value="$(arg camera_topic)" />
		<param name="down_sample_size" type="double" value="$(arg down_sample_size)"/>
		<param name="spin_rate" value="$(arg spin_rate)" />
		<param name="pc_save" value="$(arg pc_save)" />
		<param name="cylinders_topic" value="$(arg cylinders_topic)" />
		<param name="camera_frame" value="$(arg camera_frame)" />
	</node>

	<node name="desperate_rviz" pkg="rviz" type="rviz" respawn="false" args="-d $(find desperate_housewife)/launch/rviz_config.rviz" output="screen">

  </node>


</launch>

